vanilla_installer/gtk/widget-partition.ui

```diff
@@ -100,7 +100,7 @@
     <child>
       <object class="AdwPreferencesGroup" id="root_part">
         <property name="title" translatable="yes">Root Partition</property>
-        <property name="description" translatable="yes">The partition to mount under “/”. It will be automatically converted into an LVM thin pool during installation.</property>
+        <property name="description" translatable="yes">The partition to mount under “/”. It will be automatically converted into an LVM thin pool during installation. Minimum space required in the partition is 50GiB.</property>
```

Comment:
This is not correct, the root needs around 21 GB, though I'm not sure how much exactly.